### PR TITLE
ADEN-6719 Clean up MBLB leftovers

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: app/mixins/ads.js [hunk 1]
```diff
@@ -9,19 +9,13 @@ export default Mixin.create({
...
 				disableManualInsert: true
 			}
 		},
 		bottomLeaderBoard: 'BOTTOM_LEADERBOARD',
 		invisibleHighImpact: 'INVISIBLE_HIGH_IMPACT',
 		invisibleHighImpact2: 'INVISIBLE_HIGH_IMPACT_2',
-		// TODO remove in ADEN-6719
-		mobileBottomLeaderBoard: 'MOBILE_BOTTOM_LEADERBOARD',
 		mobileInContent: 'MOBILE_IN_CONTENT',
 		mobilePreFooter: 'MOBILE_PREFOOTER',
```
Comment (fraszczakszymon) [inline app/mixins/ads.js]: Should we remove MOBILE_PREFOOTER?

File: app/mixins/ads.js [hunk 2]
```diff
@@ -9,19 +9,13 @@ export default Mixin.create({
 				// ATF slot is pushed immediately (without any delay/in single request with other slots)
 				isAboveTheFold: true
 			},
-			// TODO remove in ADEN-6719
-			MOBILE_BOTTOM_LEADERBOARD: {
-				disableManualInsert: true
-			},
 			MOBILE_PREFOOTER: {
```
Comment (fraszczakszymon) [inline app/mixins/ads.js]: Should we remove MOBILE_PREFOOTER?

File: app/mixins/ads.js [hunk 3]
```diff
@@ -121,14 +109,7 @@ export default Mixin.create({
 		}
```
Comment (fraszczakszymon) [inline app/mixins/ads.js]: Should we remove MOBILE_PREFOOTER?

File: app/modules/ads.js [hunk 4]
```diff
@@ -380,7 +380,6 @@ class Ads {
 		this.slotsContext.setStatus('MOBILE_TOP_LEADERBOARD', this.isTopLeaderboardApplicable());
 		this.slotsContext.setStatus('MOBILE_IN_CONTENT', isInContentApplicable);
 		this.slotsContext.setStatus('MOBILE_PREFOOTER', this.isPrefooterApplicable(isInContentApplicable));
```
Comment (fraszczakszymon) [inline app/modules/ads.js]: Should we remove MOBILE_PREFOOTER?

File: app/mixins/ads.js [hunk 5]
```diff
@@ -97,12 +91,6 @@ export default Mixin.create({
 			adsData = this.get('adsData'),
 			globalFooter = document.querySelector('.wds-global-footer');
 
-		// TODO ADEN-6719 remove while removing MBLB leftovers
-		this.set(
-			'adsData.additionalConfig.MOBILE_PREFOOTER.disableManualInsert',
-			this.get('ads.module.adsContext.opts.isBLBOnMobileEnabled')
```
Comment (fraszczakszymon) [inline app/mixins/ads.js:103]: Is it safe to unset this flag?

File: app/mixins/ads.js [hunk 6]
```diff
@@ -148,12 +129,6 @@ export default Mixin.create({
 			trendingArticles = this.element.querySelector('.trending-articles'),
 			globalFooter = document.querySelector('.wds-global-footer');
 
-		// TODO ADEN-6719 remove while removing MBLB leftovers
-		this.set(
-			'adsData.additionalConfig.MOBILE_PREFOOTER.disableManualInsert',
-			this.get('ads.module.adsContext.opts.isBLBOnMobileEnabled')
```
Comment (fraszczakszymon) [inline app/mixins/ads.js:154]: Is it safe to unset this flag?

File: app/mixins/ads.js (hunk 5 above)
Comment (dmnsobczak) [reply app/mixins/ads.js:103]: `disableManualInsert` is hardcoded to `true` in config above
`opts.isBLBOnMobileEnabled` is set based on `wgAdDriverBottomLeaderBoardOnMobileCountries`
`wgAdDriverBottomLeaderBoardOnMobileCountries` is `[XX]` now

so `disableManualInsert` shouldn't be set to false here, so I think it is safe to remove this lines

File: app/mixins/ads.js (hunk 6 above)
Comment (dmnsobczak) [reply app/mixins/ads.js:154]: Same answer as above.

File: app/mixins/ads.js (hunk 5 above)
Comment (fraszczakszymon) [reply app/mixins/ads.js:103]: Thanks. Makes sense

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**